### DISCUSSION TITLE
Config error in oh my posh

### DISCUSSION THREAD

**Discussion by ghost** [JanDeDobbeleer/oh-my-posh#4205] [Troubleshoot] [answered]
![Screenshot (2545)](https://github.com/JanDeDobbeleer/oh-my-posh/assets/142557078/2c17f617-57fa-424a-8f72-45dc51e528b8)
how am I suppose to fix this? I tried many different options but non of them worked

shell - bash
in ubuntu under WSL version 1 windows 10

eval "$(oh-my-posh init bash)" this is what I added to .bashrc

**Comment by JanDeDobbeleer** [comment]
Can your run oh-my-posh debug? That will show what's going on.

**Reply by ghost** [reply]
Please can you tell me how to run that? 
can I know whether my computer can run oh-my-posh -  
windows 10 pro version 1903 
 OS build 18362.30
 WSL Version 1
![Screenshot (2546)](https://github.com/JanDeDobbeleer/oh-my-posh/assets/142557078/944ef933-8d77-4f1f-8766-f58fe4925b59)

**Reply by JanDeDobbeleer** [reply, marked as answer]
@qubit-hacker you paste or type `oh-my-posh debug` and press enter in that window.

**Reply by ghost** [reply]
I fixed that error. Thank you very much for your contribution! and I apologize for bothering you.

**Reply by ghost** [reply]
I just uninstalled it and reinstalled in the proper way.
